projects/project-player-library/src/lib/pages/project-details/project-details.component.html

```diff
@@ -4,7 +4,7 @@
       {{"CATEGORIES" | translate}}:
       <span>{{ getCategoryLabels() }}</span>
```

Comment:
Instead of calling getCategoryLabels() and getRecommendedFor() in the html can we store the value in the ts file and use that variable here, instead of calling the function in the html

Reply:
done